bazel/version/BUILD

```diff
@@ -0,0 +1,55 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+load("@bazel_skylib//rules:common_settings.bzl", "bool_flag", "int_flag", "string_flag")
+
+package(default_visibility = ["//toolchain/install:__pkg__"])
+
+exports_files(["gen_tmpl.py"])
+
+bool_flag(
+    name = "release",
+    build_setting_default = False,
+)
+
+string_flag(
```

Comment:
Oof, no where currently.

Where would that documentation be most discoverable do you think? Here, in `rules.bzl`, or in the `.bazelrc` where I set up aliases for them?

Reply:
I'd suggest here, and point here from bazelrc. My thinking is it might be more discoverable in bazelrc, but bazelrc will become pretty big if we keep adding features and documentation there. (plus, some things like "string or int" is self-documenting here)